pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "usn"
+version = "0.1.0"
+description = "This utility generates a netboot directory tree from an Ubuntu Server Live ISO image, an image based on the subiquity installer."
+authors = ["Taihsiang Ho (tai271828) <taihsiang.ho@canonical.com>"]
```

Comment:
@dannf Would you please add your preferable author name and email here? This line will be used as the author information of the python package.

I enabled the permission for project owner to edit the pull request directly. Editing it directly via Github UI will be a new commit appended. Or you could just let me know your preferable author information, and I will update this pull request.

Reply:
> @dannf Would you please add your preferable author name and email here? This line will be used as the author information of the python package.
> 
> I enabled the permission for project owner to edit the pull request directly. Editing it directly via Github UI will be a new commit appended. Or you could just let me know your preferable author information, and I will update this pull request.

If you don't mind updating, it's dann frazier <dannf@ubuntu.com>

Reply:
I don't mind. Will do!